kogito-usertasks-with-security-quarkus/src/main/resources/application.properties

```diff
@@ -7,6 +7,6 @@ quarkus.security.users.embedded.roles.john=employees
 quarkus.security.users.embedded.users.mary=mary
 quarkus.security.users.embedded.roles.mary=managers
 quarkus.security.users.embedded.users.poul=poul
-quarkus.security.users.embedded.roles.poul=interns
+quarkus.security.users.embedded.roles.poul=interns,managers
```

Comment:
why this change here?

Reply:
@mswiderski the intention here was make the readme a little more consistent.. Regarding the process requires 2 managers approval and the process is checking they are not the same. Even passing 'user=john&group=managers' to the secondLineApproval would work too, we can keep it as it was :)